scripts/jenkins/build.sh

```diff
@@ -21,21 +21,21 @@ rm -f angular.js.size
 
 
 # BUILD #
-yarn run grunt -- ci-checks package --no-color
```

Comment:
❤️

Reply:
That reminds me I can shorten it to `yarn grunt` instead of `yarn run grunt`... We already use the former form in a few places.